app/models/admin-topojson.js

```diff
@@ -0,0 +1,40 @@
+var mongoose = require('mongoose');
+
+// TODO(jetpack): administrative regions can change. think about the update
+// story. maybe just created/updated_at fields?
+
+var admin_topojson_schema = new mongoose.Schema({
+  // ISO 3166-1 alpha-2 two letter country code.
+  country_code: {
+    type: String,
+    index: true,
+    validate: {
+      validator: function(v) {
+        return /[a-z]{2}/.test(v);
+      },
+      message: '{VALUE} is not a valid name!'
+    }
+  },
+
+  // Retention percentage that was given to topojson.simplify() call.
+  simplification: {
+    type: Number,
+    required: true,
+    index: true
```

Comment:
Do we need an index here? We probably won't have that many simplification levels per country.